### ISSUE TITLE
Location of conda env directory

### ISSUE THREAD

**Issue by jmtsuji** [jmtsuji/rotary#62]
I'm testing out the CLI for the first time (nice work!) and am making some Github issues for things I notice during the process.

## Environment
Rotary end_repair commit [9a4678b](https://github.com/jmtsuji/rotary/commit/9a4678b9a03865c2f295079feedc368139fce0a2)
Running on a Linux (Ubuntu) server

## Problem description
Currently, in `rotary.py`, the `conda_env_directory` is hard-coded to be within the output directory:

[Approx. line 99](https://github.com/jmtsuji/rotary/blob/9a4678b9a03865c2f295079feedc368139fce0a2/rotary/rotary.py#L99C1-L99C1)
```python
    conda_env_directory = os.path.join(output_dir_path, 'conda_env')
    os.makedirs(conda_env_directory, exist_ok=True)
```

This is nice because it keeps the settings for a single run within the output folder. However, because there are a lot of installations in the pipeline, the time (and storage space) needed to keep making new conda installs for each rotary run could become substantial.

## Proposed solution
What do you think about setting the 'conda_env' directory to be within the DB dir? This is the approach taken by ATLAS. Although it could potentially lead to permissions issues across multiple users, it will keep the installs centralized and reusable between runs.

We could also consider exposing the conda_env_directory as a flag, but I am not sure how important this is -- I'm not sure how many users will want to change the storage location.

Thanks!

**Comment by LeeBergstrand** [comment]
@jmtsuji I was just copying what you had done in your original build instructions in the ```README.md```. However, moving the Conda Env to the db_dir was a suggestion that I was going to make.

**Comment by LeeBergstrand** [comment]
Addressed in https://github.com/jmtsuji/rotary/pull/64

**Comment by jmtsuji** [comment]
> @jmtsuji I was just copying what you had done in your original build instructions in the README.md. However, moving the Conda Env to the db_dir was a suggestion that I was going to make.

@LeeBergstrand Makes sense -- I simplified the instructions in the original README because it was confusing when the user had to manually specify where the conda_env folder went. Glad to hear you agree that moving the conda_env dir to the db_dir is something you also think is a good idea. I approved the PR, so feel free to close this issue once you merge!
